.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+- '3.7'
+services:
+- docker
+before_script:
+- pip install docker-compose
+- docker build --target production --tag todo-app:dev .
+- docker run -d -p 127.0.0.1:5000:5000 --env-file .env todo-app:dev
+- docker ps -a
```

Comment:
Why are you running your production container? In detached mode we won't even see if this succeeds/fails, while in non-detached it would block our pipeline. We should instead focus on running our tests here